script/upgrades/FX01/deploy/FX01-00-Create-Proxies.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Hmm I think we didn't sync properly on this. I also needed the proxy addresses so I also created script to deploy them [here](https://github.com/mento-protocol/mento-deployment/blob/main/script/upgrades/G7Whitelist/deploy/G7-00-Deploy-Proxys.sol). You can load them as `contracts.loadSilent("G7-00-Deploy-Proxys", "latest");` I already wrote down those addresses in the CGP and forum post for the whitelisting proposal.

Next time we can put this in a separate PR so that we can both reference it